src/thread_pool.rs

```diff
@@ -109,14 +113,18 @@ impl Registry {
         self.work_available.notify_all();
     }
 
-    pub unsafe fn inject(&self, injected_jobs: &[*mut Job<LockLatch>]) {
+    pub unsafe fn inject(&self, injected_jobs: &[*mut (Job<LockLatch> + 'static)]) {
```

Comment:
Huh. I'm a bit surprised, but I guess I forget what the defaulting rules are around `*mut`.